package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oc-client-browser",
-  "version": "1.3.2",
+  "version": "1.3.4",
```

Comment:
a note here, the version is changed when publishing. I'll revert this.

Reply:
Thanks sorry

Reply:
Wait. I thought I only bumped it by one... :confused:

Reply:
no worries. Published 1.3.3